client/tasks/fills/PaymentGatewaySuggestions/components/WCPay/Suggestion.js

```diff
@@ -47,6 +49,14 @@ export const Suggestion = ( { paymentGateway, onSetupCallback = null } ) => {
 		installed: isInstalled,
 	} = paymentGateway;
 
+	const { createNotice } = useDispatch( 'core/notices' );
+
+	if ( installed && onSetupCallback === null ) {
```

Comment:
@moon0326 Could you add a comment to explain why we add this? I found it took me a few minutes to understand how this works. I think this will be helpful when others work on this file. Thanks!

Reply:
@chihsuan My apologies. Added comment in https://github.com/woocommerce/woocommerce-admin/pull/8367/commits/1b8de38459b8b9d25822f74c32c0c79353b91768 👍

Reply:
No worries! Looks good, thank you. 🚢